ivy/functional/backends/tensorflow/linear_algebra.py

```diff
@@ -186,8 +186,6 @@ def matrix_norm(
             ret = x
         else:
             ret = tf.reduce_sum(tf.linalg.svd(x, compute_uv=False), axis=-1)
-    elif ord == "fro":
```

Comment:
we need to support "fro" as shown [here](https://data-apis.org/array-api/2021.12/extensions/generated/signatures.linalg.matrix_norm.html?highlight=matrix_norm).

Reply:
Actually, the "fro" is still supported here. Since when the variable ord equals "fro", it will still be passed to tf.linalg.norm in line 190 (latest commit).
In the previous commit, when ord equals "fro", see line 190 (previous commit):
`
ret = tf.linalg.norm(x, 2, axes, keepdims)
` 
the second positional arg is for ord, but it uses a constant value 2.

Reply:
ah yes, thanks for explaining!